.travis.yml

```diff
@@ -28,17 +27,19 @@ script:
 
 deploy:
   - provider: pages
-    github-token: $GITHUB_TOKEN
-    #keep-history: true
+    token: $GITHUB_TOKEN
+    keep_history: false
     skip_cleanup: true
-    committer-from-gh: true
-    target-branch: gh-pages
-    local-dir: html-report
+    committer_from_gh: true
+    target_branch: gh-pages
+    local_dir: html-report
+    strategy: git
     on:
       all_branches: true
       condition: ${DISTRO} =~ ^fedora.*$
   - provider: script
     script: ./docker-build --verbose --config .build.yml --release github
+    skip_cleanup: true
```

Comment:
In previous commit i removed `skip_cleanup: true` like you did for mate-common.
Was this wrong?